openslides/users/static/js/users/base.js

```diff
@@ -133,7 +134,7 @@ angular.module('OpenSlidesApp.users', [])
                         addition.push(structure_level);
                     }
                     if (number) {
-                        addition.push(number);
+                        addition.push(gettextCatalog.getString('No.') + ' ' + number);
```

Comment:
Here we need a context description for the translaters to show that this is an abbrevation of number and not the opposite of yes.